queries.sql

```diff
@@ -8,4 +8,60 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'yes';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
+ 
+  UPDATE animals SET species = 'unspecified';
+ 
+ROLLBACK TRANSACTION;
+
+BEGIN TRANSACTION;
+
+  UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+  UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+
+COMMIT TRANSACTION;
+
+  SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+
+  DELETE FROM animals ;
+
+ROLLBACK TRANSACTION;
+
+  SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+
+  DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+  SAVEPOINT transaction;
+
+  UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TRANSACTION TO SAVEPOINT transaction;
+
+  UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+
+COMMIT TRANSACTION;
+
+-- How many animals are there?
+SELECT COUNT(*) AS count_of_animals FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(*) AS count_of_not_escapeds FROM animals WHERE escape_attempts > 1;
```

Comment:
- [ ] You are selecting animals where escape attempts more than 1. You should find animals with **no escape attempts**

Reply:
This issue is fixed!